### PR TITLE
[Update] Deploy WordPress through the Linode Marketplace

### PR DESCRIPTION
**Pull request by timzl**
Clarified instructions on file path for password reset.

### CODE REVIEW

File: docs/products/tools/marketplace/guides/wordpress/index.md [hunk 1]
```diff
@@ -116,10 +116,10 @@ If you need to reset your admin user's password and you aren't receiving the pas
 
 1. Log in to the Compute Instance using [SSH](/docs/guides/connect-to-server-over-ssh/) or [Lish](/docs/products/compute/compute-instances/guides/lish/).
 
-1.  Navigate to the directory of your WordPress installation:.
+1.  Navigate to the `public_html` directory of your WordPress installation:.
 
     ```command
-    cd /var/www/wordpress
+    cd /var/www/[domain]/public_html
```
Comment (Rajakavitha1) [inline docs/products/tools/marketplace/guides/wordpress/index.md:122]:
```suggestion
    cd /var/www/example.com/public_html
```
Please see, https://www.linode.com/docs/guides/linode-writers-formatting-guide/#placeholders

**Review by Rajakavitha1 — CHANGES_REQUESTED** [review]

File: docs/products/tools/marketplace/guides/wordpress/index.md (hunk 1 above)
Comment (Rajakavitha1) [reply docs/products/tools/marketplace/guides/wordpress/index.md:122]: Thank you for the PR @timzl

**Review by wildmanonline — APPROVED** [review]

**Review by wildmanonline — APPROVED** [review]

**Comment by wildmanonline** [comment]
@Rajakavitha1 it appears that we are blocked from merging this. Can you approve the change that you requested?

**Review by Rajakavitha1 — APPROVED** [review]
LGTM!

**Pull request merged**